src/main/java/com/test/sb/service/ProductService.java

```diff
@@ -0,0 +1,20 @@
+package com.test.sb.service;
+
+import com.test.sb.model.Product;
+import com.test.sb.repository.ProductRepository;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProductService implements GeneralService<Product> {
```

Comment:
```suggestion
public interface ProductService extends GeneralService<Product> {
```
and then we can create class ProductServiceImpl
same for other services